### PR TITLE
Add dist/ directory back to modules.adonis [AD-1317]

### PR DESCRIPTION
**Pull request by sftwninja**
Turns out you need that directory. ¯\\\_(ツ)\_/¯

AD-1317

### CODE REVIEW

File: dist/src/pipes/phone.pipe.spec.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export {};
```
Comment (julamula) [inline dist/src/pipes/phone.pipe.spec.d.ts]: what is the "d" in the file name?

Comment (sftwninja) [reply dist/src/pipes/phone.pipe.spec.d.ts]: https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html

It's autogenerated by typescript for IDEs/compiler to generate types to link against during compilation.

**Comment by sftwninja** [comment]
@julamula intentional since it's an import-able NPM module.

File: dist/src/pipes/phone.pipe.spec.d.ts (hunk 1 above)
Comment (julamula) [reply dist/src/pipes/phone.pipe.spec.d.ts]: cool

**Pull request merged**